suites/message/nested.go

```diff
@@ -79,7 +81,7 @@ func MessageTest_NestedSends(t *testing.T, factory state.Factories) {
 		result := stage.sendOk(newAddr, amtSent, builtin.MethodSend, nil, nonce)
 
 		td.AssertBalance(stage.msAddr, big.Sub(multisigBalance, amtSent))
-		td.AssertBalance(stage.creator, big.Sub(balanceBefore, result.Receipt.GasUsed.Big()))
+		td.AssertActorChange(stage.creator, balanceBefore, result.Msg.GasLimit, result.Msg.GasPrice, result.Msg.Value, result.Receipt, result.Msg.CallSeqNum+1)
```

Comment:
Here and in many below you've added unnecessary checking of the call sequence number, making this change look more verbose than necessary. Add an `AssertSenderBalance` that's just `AssertBalance` with the gas/value parameters but without nonce. In fact you probably want that almost all the time, and could revert AssertActor.